project/Dependencies.scala

```diff
@@ -15,10 +15,12 @@ object Dependencies {
   lazy val loggingVersion = "3.9.0"
   lazy val slf4jVersion = "1.7.25"
   lazy val awsSdkVersion = "1.11.349"
-  val tensorflowVersion = "1.11.0"
+  val tensorflowVersion = "0.3.1" // Match Tensorflow 2.4.1 https://github.com/tensorflow/java/#tensorflow-version-support
```

Comment:
Hmm maybe we should rename this to `tensorflowJavaVersion` to make it more clear that it's a different package